rules-reviewed/eap7/eap71/hibernate51-53.windup.xml

```diff
@@ -119,17 +119,23 @@
             </perform>
         </rule>
         -->
-        <!--https://issues.jboss.org/browse/WINDUPRULE-387
+        <!--https://issues.jboss.org/browse/WINDUPRULE-387-->
         <rule id="hibernate51-53-01100">
             <when>
+                <and>
+                    <javaclass references="javax.persistence.TableGenerator">
+                        <location>IMPORT</location>
+                    </javaclass>
+                </and>
             </when>
             <perform>
+                <hint title="Hibernate 5.3  - @TableGenerator changes interpretation" effort="1" category-id="potential">
```

Comment:
```suggestion
                <hint title="Hibernate 5.3  - @TableGenerator changes interpretation" effort="1" category-id="mandatory">
```
it seems to me that this is `mandatory` because documentation says "must set [...] to `false`"

Reply:
I put "potential" as you suggested so : https://issues.jboss.org/browse/WINDUPRULE-387?focusedCommentId=13698974&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13698974

Reply:
You're totally right and, tbh, i'm little bit scared from myself and my memory now :sob: 
Let's keep this `potential` :+1: